examples/functions.js

```diff
@@ -14,6 +14,9 @@ var data = {
   ]
 };
 
-var ret = ejs.compile(read(path, 'utf8'), {filename: path})(data);
+var ret = ejs.compile(read(path, 'utf8'), {
+    filename: path,
+    outputFunctionName: 'echo'
```

Comment:
Same feedback as above -- since this is not likely the default use-case, it make make more sense to make this a separate example.